### ISSUE TITLE
Copy static folder for each theme

### ISSUE THREAD

**Issue by ChristopherBiscardi** [ChristopherBiscardi/gatsby-theme-examples#7]
The static folder in each theme should be copied into the final static dir. Right not only the user's is.

**Comment by ChristopherBiscardi** [comment]
Dependent on #9 because that includes the code to fetch the full list of themes so we can dig into them and find their static folders.

**Comment by ChristopherBiscardi** [comment]
This happens when building now. We need another PR to address develop-time
